CMakeLists.txt

```diff
@@ -89,8 +89,12 @@ if ("${isSystemDir}" STREQUAL "-1")
 endif ("${isSystemDir}" STREQUAL "-1")
 set (CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
-set(INSTALL_PATH "/opt/prplmesh")
-set(TMP_PATH "/tmp/beerocks")
+if(NOT DEFINED INSTALL_PATH)
+    set(INSTALL_PATH "/opt/prplmesh")
+endif()
+if(NOT DEFINED TMP_PATH)
+    set(TMP_PATH "/tmp/beerocks")
+endif()
```

Comment:
Again, use a cache variable instead. However, why do you need to override this at all for RDKB?

Reply:
I rechecked that it's not needed to redefine this variable in recipe.
Reverted my changes for it.